cppcache/src/ThinClientRegion.cpp

```diff
@@ -2772,7 +2774,7 @@ void ThinClientRegion::receiveNotification(TcrMessage* msg) {
   }
 
   lock.unlock();
-  if (TcrMessage::getAllEPDisMess() != msg) _GEODE_SAFE_DELETE(msg);
+  _GEODE_SAFE_DELETE(msg);
```

Comment:
Any easy opportunity to make `msg` a smart pointer?

Reply:
This one was a little larger than I anticipated, but I think it's a really nice refactor.